src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -67,14 +68,14 @@
               [:wallet-send-input-amount stack-id]
               [:wallet-select-asset stack-id])]]})))
 
-(rf/reg-event-fx :wallet/send-select-token
+(rf/reg-event-fx
+ :wallet/send-select-token
  (fn [{:keys [db]} [{:keys [token stack-id]}]]
    {:db (assoc-in db [:wallet :ui :send :token] token)
-    :fx [[:dispatch-later
-          {:ms       1
-           :dispatch [:navigate-to-within-stack [:wallet-send-input-amount stack-id]]}]]}))
+    :fx [[:navigate-to-within-stack [:wallet-send-input-amount stack-id]]]}))
```

Comment:
Thanks @ulisesmac for warning about this PR and discussion! We might need to update DB when navigating, so fx might not be what we want here. This would be replaced with a `dispatch` if we merge #18593, so better to reach to a common agreement before merging any of these PRs to avoid unnecessarily reverting changes. Context: https://github.com/status-im/status-mobile/pull/18593#discussion_r1468103430

Reply:
Also, another topic, but it is confusing to have navigation effects and events with same naming, like https://github.com/status-im/status-mobile/blob/972a88e7ccc9fd5fc697caa42919079a4da73859/src/status_im/navigation/events.cljs#L55 and https://github.com/status-im/status-mobile/blob/602b27105bcc4960a8748dd60af6bdc8967559c0/src/status_im/navigation/effects.cljs#L91
.......
or
.......

https://github.com/status-im/status-mobile/blob/972a88e7ccc9fd5fc697caa42919079a4da73859/src/status_im/navigation/events.cljs#L34 and https://github.com/status-im/status-mobile/blob/602b27105bcc4960a8748dd60af6bdc8967559c0/src/status_im/navigation/effects.cljs#L78

We should rename events to have navigation / namespace prefix as per our guidelines https://github.com/status-im/status-mobile/blob/5cc027a1f58a35f4b7980612a9080a621d134d6f/doc/new-guidelines.md#L364-L372